anybox/recipe/odoo/runtime/session.py

```diff
@@ -7,29 +7,20 @@
 
 
 try:
-    import openerp as odoo
+    import odoo
 except ImportError:
     try:
-        import odoo
+        import openerp as odoo
     except ImportError:
         warnings.warn("This must be imported with a buildout odoo recipe "
                       "driven sys.path", RuntimeWarning)
-    else:
-        try:
-            from odoo.cli import server as startup
-        except ImportError:
-            from .backports.cli import server as startup
-        from odoo.tools import config
-        from odoo import SUPERUSER_ID
-        from odoo.tools.parse_version import parse_version
-else:
-    try:
-        from openerp.cli import server as startup
-    except ImportError:
-        from .backports.cli import server as startup
-    from openerp.tools import config
-    from openerp import SUPERUSER_ID
-    from openerp.tools.parse_version import parse_version
+try:
+    from odoo.cli import server as startup
+except ImportError:
+    from .backports.cli import server as startup
+from odoo.tools import config
```

Comment:
Thanks, I've test something similar, looks pretty much cleaner however this code result to make integration test failing on v8

```
pverkest@petrus-v:~/tmp/aro_8$ bin/nosetests -d aro8 -- -s -v ../aro/tests_with_odoo/
Traceback (most recent call last):
  File "bin/nosetests", line 60, in <module>
    from anybox.recipe.odoo.runtime.session import Session
  File "/home/pverkest/tmp/aro/anybox/recipe/odoo/runtime/session.py", line 21, in <module>
    from odoo.tools import config
ImportError: No module named tools
```

This sounds strange to me, it's like ``import odoo`` do not raise ImportError exceptions... I'll go in deep on that...